string request
geometry_msgs/Pose artur_scan_pose
